<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerDialog</class>
 <widget class="QWidget" name="TimerDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Timer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radio_stop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="spin_min"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_sec">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>sec</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_start</tabstop>
  <tabstop>radio_stop</tabstop>
  <tabstop>spin_min</tabstop>
  <tabstop>spin_sec</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_stop</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_stop</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_sec</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
